cli/cmd/compose/convert.go

```diff
@@ -76,9 +89,14 @@ func runConvert(ctx context.Context, opts convertOptions, services []string) err
 	if err != nil {
 		return err
 	}
+	var out io.Writer = os.Stdout
 	if opts.Output != "" {
```

Comment:
Should this still work with quiet mode? The help says that it only does validation

Reply:
Indeed. --ouput might be used with a path and then we would override file, and not just "validate and quit".
fixed